apertium-get.py

```
@@ -0,0 +1,313 @@
+        epilog='''EXAMPLES
+       apertium-get nno-nob
+
+       Download and set up apertium-nno-nob, along with its nno and
+       nob dependencies.
+
+       sudo apt-get install giella-sme
+       apertium-get -x sme sme-nob
+
+       Install giella-sme through apt-get, then install download and set
+       up apertium-sme-nob, along with the nob dependency (but not sme).
+
+       apertium-get -l trunk
+
+       List available language pairs in SVN trunk.
+
+       apertium-get -l | grep kaz
+
+       List available language pairs involving Kazakh.''', formatter_class=argparse.RawDescriptionHelpFormatter)
+    parser.add_argument('-l', '--list', nargs='*', choices=pmodules, help='list available pairs in MODULES instead of setting up data. If no modules are specified, all pairs will be listed.', metavar='MODULES')
+    parser.add_argument('-m', '--modules', nargs='*', choices=lmodules, help='list available pairs in MODULES instead of setting up data. If no modules are specified, all pairs will be listed.', metavar='MODULES')
+    parser.add_argument('-k', '--keep-going', action='store_true', help='keep going even if a dependency fails.')
+    parser.add_argument('-s', '--skip-update', action='store_true', help="don't rebuild up-to-date dependencies/pairs")
+    parser.add_argument('-x', '--skip', action='append', nargs=1, help='skip data dependency DEP (useful if DEP is installed through a package manager); may be specified multiple times')
+    parser.add_argument('-d', '--depth', nargs=1, type=int, default=0, help="specify a --depth to 'git clone'")
+    parser.add_argument('pairs', nargs='*', help='pairs or modules to install')
+    args = parser.parse_args()
+
+    get_urls()
+
+    if args.list != None:
+        list_pairs(args.list)
+    elif args.modules != None:
+        list_language_modules(args.modules)
+    else:
+        if len(args.pairs) == 0:
+            print('ERROR: No language pair specified.\n')
```

you can raise an argparse error, that'll do all this stuff for you